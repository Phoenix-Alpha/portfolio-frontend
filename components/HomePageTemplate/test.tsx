import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { HomePageTemplate } from '.'

describe('<HomePageTemplate />', () => {
  it('should renders correct <HomePageTemplate />', () => {
    const wrapper = shallow(<HomePageTemplate />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
