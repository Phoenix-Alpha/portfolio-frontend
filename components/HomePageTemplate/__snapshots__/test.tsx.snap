// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePageTemplate /> should renders correct <HomePageTemplate /> 1`] = `
<Fragment>
  <HomeHeroSection />
  <HomeFeaturedPostsSection />
  <HomeFeaturedProjectsSection />
</Fragment>
`;
